setup.cfg

```diff
@@ -0,0 +1,5 @@
+[build_sphinx]
+all-files = 1
+source-dir = docs
+build-dir = docs/_build/
+warning-is-error = 0
```

Comment:
This should be 1 for production use.

Fix sphinx until everything works.

Reply:
Noted :)